DSharpPlus/Entities/Application/DiscordApplication.cs

```diff
@@ -148,10 +148,11 @@ public string GetAvatarUrl(ImageFormat fmt, ushort size = 1024)
         /// <summary>
         /// Retrieves this application's assets.
         /// </summary>
+        /// <param name="useCacheIfPossible">True to return cached assets if possible, false to always retrieve fresh assets from Discord.</param>
         /// <returns>This application's assets.</returns>
-        public async Task<IReadOnlyList<DiscordApplicationAsset>> GetAssetsAsync()
+        public async Task<IReadOnlyList<DiscordApplicationAsset>> GetAssetsAsync(bool useCacheIfPossible = true)
```

Comment:
This contrasts with the behavior of the `GetUserAsync` method, defined [here](https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus/Clients/DiscordClient.cs#L399)